client/my-sites/email/paths.js

```diff
@@ -171,11 +172,15 @@ export function emailManagementEdit(
 
 export function emailManagementInbox( siteName = null ) {
 	if ( siteName ) {
-		return `${ emailManagementPrefix }/inbox/${ siteName }`;
+		return `${ emailManagementInboxPrefix }/${ siteName }`;
 	}
-	return `${ emailManagementPrefix }/inbox`;
+	return `${ emailManagementInboxPrefix }/`;
 }
 
 export function isUnderEmailManagementAll( path ) {
 	return path?.startsWith( emailManagementAllSitesPrefix + '/' );
 }
+
+export function isUnderEmailInbox( path ) {
```

Comment:
Shouldn't we use a name that better matches the path?

```suggestion
export function isUnderEmailManagementInbox( path ) {
```

Reply:
Yes, thank you :)